src/infrastructure/server/middlewares/authorize.ts

```diff
@@ -36,17 +37,22 @@ class AuthorizationMiddleware {
 
     let currentUser: User | null = null
 
-    if (accessToken) {
-      const googleUser = await this.googleAuthService.getCurrentUser(
-        accessToken
-      )
-      currentUser = await this.userRepository.findByEmail(googleUser.email)
+    try {
```

Comment:
I guess we should set up integration tests for testing middlewares that are coupled to `Koa`